### DISCUSSION TITLE
Welcome to Nemo Gradebook Discussions!

### DISCUSSION THREAD

**Discussion by andrewpbray** [gradebook-dev/gradebook-app#109] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, please comment below with two items:
  
  1. A screenshot of your grading policy from your syllabus.
  2. A screenshot of your grading policy as it appears in Nemo Gradebook.

**Comment by joshuagrossman** [comment]
Gradebook is awesome!!

One question (for now):
- To encourage everyone to complete all of the HWs, I floor my lowest HW grade at 80%. It's similar to a drop, but it nominally rewards students who complete all of the assignments. Is it possible to implement something like this on the Gradebook end?
